src/components/toolbox/passphraseInput/passphraseInput.css

```diff
@@ -1,24 +1,72 @@
 @import '../../../app/variables.css';
 @import '../../../app/mixins.css';
 
+.wrapper {
+  margin: 0;
+  position: relative;
+}
+
 .inputs {
+  width: 100%;
+  display: flex;
   flex-wrap: wrap;
+  flex-direction: row;
+  border: 1px solid var(--dark-border-color);
+  border-radius: 3px;
+  background-color: var(--color-white);
+  padding: 0px 24px;
+  box-sizing: border-box;
+  outline: none;
+
+  &:focus-within {
+    border-color: var(--color-ultramarine-blue);
+  }
 
   & > span {
```

Comment:
I know it's not introduced in this PR, but https://github.com/LiskHQ/lisk-hub/blame/development/docs/CSS_GUIDE.md#L15

Reply:
not resolved yet